apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Snippets/SpannerConnectionSnippets.cs

```diff
@@ -424,16 +424,46 @@ public async Task WhenSessionPoolReady()
             // End sample
         }
 
-        [Fact]
-        public void GetSessionPoolDatabaseStatistics()
+        [Fact]
+        public void GetSessionPoolStatistics()
+        {
+            string connectionString = _fixture.ConnectionString;
+
+            // Sample: GetSessionPoolDatabaseStatistics
+            // Additional: GetSessionPoolDatabaseStatistics
+            using (SpannerConnection connection = new SpannerConnection(connectionString))
+            {
+#pragma warning disable CS0618 // Type or member is obsolete

```

Comment:
Let's update the samples to use the non-deprecated methods instead.

And please check the docs where these samples are included in case they need updating too. Ping me later to tell you how to find the docs where these samples are being shown.